dts/bindings/tach/nuvoton,npcx-tach.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2021 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nuvoton, NPCX-Tachometer node
+
+compatible: "nuvoton,npcx-tach"
+
+include: tach.yaml
+
+properties:
+    reg:
+        required: true
+    clocks:
+        required: true
+    pinctrl-0:
+        type: phandles
+        required: false
+        description: configurations of pinmux controllers in tachometers
+    sample_clk:
+        type: int
+        required: false
+        description: sampling clock frequency of tachometer
```

Comment:
Note that this must be set to 32768 when using the low-frequency clock (LFCLK).

Reply:
Done.